test/tests/p5.Metro.js

```diff
@@ -0,0 +1,85 @@
+const expect = chai.expect;
+let metro;
+
+describe('p5.Metro', function () {
+  beforeEach(function () {
+    metro = new p5.Metro();
+  });
```

Comment:
it might help with garbage collection to add an `afterEach` to call the "dispose" method which removes references to the Web Audio nodes freeing up those resources for garbage collection.

But if things are running just fine so far, maybe that's not necessary.

Reply:
I am surprised to find out that metro does not have a dispose function. What should I do?
may be clear out the variable?

Reply:
ah, that sounds like an issue that should be handled in a separate PR, then!